src/theme/qml/ComboBox.qml

```diff
@@ -31,24 +31,25 @@ T.ComboBox {
     id: control
 
     implicitWidth: Math.max(background ? background.implicitWidth : 0,
-                            contentItem.implicitWidth) + leftPadding + rightPadding + spacing + indicator.implicitWidth + rightPadding
+                            contentItem.implicitWidth + leftPadding + rightPadding + spacing + indicator.implicitWidth + rightPadding)
```

Comment:
Correct one is: `contentItem.implicitWidth + leftPadding + rightPadding) + spacing + indicator.implicitWidth + rightPadding`